projects/mercury/package.json

```diff
@@ -57,7 +57,8 @@
     "text-table": "^0.2.0",
     "typeface-roboto": "^1.1.13",
     "use-deep-compare-effect": "^1.3.0",
-    "webdav": "^4.0.0"
+    "webdav": "^4.0.0",
+    "webpack": "^5.75.0"
```

Comment:
webpack is a dependency of e.g. react-scripts, so it will be fetched anyway and there will be version conflicts. It shouldn't be necessary to specify it here, unless you had some issues with this? If so, do you remember what was the problem?

Reply:
This was automatically updated, I didn't do it manually.

Reply:
Could you remove it? It must have been added by an install command that you ran locally.